--- gist-1.0.3/gist~	2010-03-01 09:29:22.000000000 +0200
+++ gist-1.0.3/gist	2010-03-22 15:22:24.915526455 +0200
@@ -174,11 +174,11 @@
 
   def copy(content)
     cmd = case true
-    when system("which pbcopy &> /dev/null")
+    when system("which pbcopy > /dev/null 2>&1")
       :pbcopy
-    when system("which xclip &> /dev/null")
+    when system("which xclip > /dev/null 2>&1")
       :xclip
-    when system("which putclip &> /dev/null")
+    when system("which putclip > /dev/null 2>&1")
       :putclip
     end
 
